cmd/firesol/cli/reader-node-stdin.go

```diff
@@ -20,49 +20,49 @@ import (
-			mergeArchiveStoreURL := MustReplaceDataDir(sfDataDir, viper.GetString("common-blocks-store-url"))
+			archiveStoreURL := MustReplaceDataDir(sfDataDir, viper.GetString("common-one-block-store-url"))
+			mergeArchiveStoreURL := MustReplaceDataDir(sfDataDir, viper.GetString("common-merged-blocks-store-url"))
 
 			consoleReaderFactory := getConsoleReaderFactory(
 				appLogger,
-				viper.GetString("mindreader-node-deepmind-batch-files-path"),
-				viper.GetBool("mindreader-node-purge-account-data"),
+				viper.GetString("reader-node-firehose-batch-files-path"),
+				viper.GetBool("reader-node-purge-account-data"),
 			)
-			metricID := "mindreader-node-stdin"
+			metricID := "reader-node-stdin"
 			headBlockTimeDrift := metrics.NewHeadBlockTimeDrift(metricID)
 			headBlockNumber := metrics.NewHeadBlockNumber(metricID)
-			metricsAndReadinessManager := nodeManager.NewMetricsAndReadinessManager(headBlockTimeDrift, headBlockNumber, viper.GetDuration("mindreader-node-readiness-max-latency"))
+			metricsAndReadinessManager := nodeManager.NewMetricsAndReadinessManager(headBlockTimeDrift, headBlockNumber, viper.GetDuration("reader-node-readiness-max-latency"))
 
-			return nodeMindreaderStdinApp.New(&nodeMindreaderStdinApp.Config{
-				GRPCAddr:                     viper.GetString("mindreader-node-grpc-listen-addr"),
+			return nodeReaderStdinApp.New(&nodeReaderStdinApp.Config{
+				GRPCAddr:                     viper.GetString("reader-node-grpc-listen-addr"),
 				ArchiveStoreURL:              archiveStoreURL,
 				MergeArchiveStoreURL:         mergeArchiveStoreURL,
-				MergeThresholdBlockAge:       viper.GetString("mindreader-node-merge-threshold-block-age"),
-				MindReadBlocksChanCapacity:   viper.GetInt("mindreader-node-blocks-chan-capacity"),
-				StartBlockNum:                viper.GetUint64("mindreader-node-start-block-num"),
-				StopBlockNum:                 viper.GetUint64("mindreader-node-stop-block-num"),
-				WorkingDir:                   MustReplaceDataDir(sfDataDir, viper.GetString("mindreader-node-working-dir")),
-				WaitUploadCompleteOnShutdown: viper.GetDuration("mindreader-node-wait-upload-complete-on-shutdown"),
-				OneblockSuffix:               viper.GetString("mindreader-node-oneblock-suffix"),
-				LogToZap:                     viper.GetBool("mindreader-node-log-to-zap"),
-				DebugDeepMind:                viper.GetBool("mindreader-node-debug-deep-mind"),
-			}, &nodeMindreaderStdinApp.Modules{
+				MergeThresholdBlockAge:       viper.GetString("reader-node-merge-threshold-block-age"),
+				MindReadBlocksChanCapacity:   viper.GetInt("reader-node-blocks-chan-capacity"),
+				StartBlockNum:                viper.GetUint64("reader-node-start-block-num"),
+				StopBlockNum:                 viper.GetUint64("reader-node-stop-block-num"),
+				WorkingDir:                   MustReplaceDataDir(sfDataDir, viper.GetString("reader-node-working-dir")),
```

Comment:
Let's also replace any occurrences of `sfDataDir` to become  just `dataDir`. Please make sure the default value for `sf-data` is changed to be `firedata`.